sql/core/src/test/scala/org/apache/spark/sql/CharVarcharTestSuite.scala

```diff
@@ -843,16 +843,6 @@ class FileSourceCharVarcharTestSuite extends CharVarcharTestSuite with SharedSpa
     }
   }
 
-  // TODO(SPARK-33898): Move these tests to super after SHOW CREATE TABLE for v2 implemented
```

Comment:
The TODO says we should move it to the super class `CharVarcharTestSuite`, not `ShowCreateTableSuiteBase`

Reply:
Could you help me review the [#34773](https://github.com/apache/spark/pull/34773). In the PR. I try to fix two problem 1.Move the SHOW CREATE TABLE w/ char/varchar to CharVarcharDDLTestBase
2.Fix the behavior different with v1 command that about the TBLPROPERTIES
Thank you very much. @cloud-fan